packages/frontend/src/components/RefreshDbtButton/index.tsx

```diff
@@ -1,48 +1,84 @@
-import { DbtProjectType, ProjectType } from '@lightdash/common';
-import { Anchor } from '@mantine/core';
-import React, { ComponentProps, FC } from 'react';
+import { DbtProjectType, JobStatusType, ProjectType } from '@lightdash/common';
+import {
+    Anchor,
+    Badge,
+    Box,
+    Button,
+    Popover,
+    Text,
+    Tooltip,
+} from '@mantine/core';
+import { IconRefresh } from '@tabler/icons-react';
+import { useEffect, useState } from 'react';
 import { useParams } from 'react-router-dom';
 import { useProject } from '../../hooks/useProject';
 import { useRefreshServer } from '../../hooks/useRefreshServer';
 import { useActiveJob } from '../../providers/ActiveJobProvider';
 import { useApp } from '../../providers/AppProvider';
 import { useTracking } from '../../providers/TrackingProvider';
 import { EventName } from '../../types/Events';
-import { BigButton } from '../common/BigButton';
-import {
-    DisabledRefreshDbt,
-    LoadingSpinner,
-    PreviewTag,
-    RefreshDbt,
-} from './RefreshDbtbutton.styles';
+import MantineIcon from '../common/MantineIcon';
 
-const RefreshDbtButton: FC<ComponentProps<typeof BigButton>> = (props) => {
+const RefreshDbtButton = () => {
     const { projectUuid } = useParams<{ projectUuid: string }>();
     const { data } = useProject(projectUuid);
     const { activeJob } = useActiveJob();
-    const { mutate } = useRefreshServer();
-    const isLoading = activeJob && activeJob?.jobStatus === 'RUNNING';
+    const { mutate: refreshDbtServer } = useRefreshServer();
+    const [isLoading, setIsLoading] = useState(false);
 
     const { track } = useTracking();
     const { user } = useApp();
 
+    useEffect(() => {
+        if (activeJob) {
+            if (
+                [JobStatusType.STARTED, JobStatusType.RUNNING].includes(
+                    activeJob.jobStatus,
+                )
+            ) {
+                setIsLoading(true);
+            }
+
+            if (
+                [JobStatusType.DONE, JobStatusType.ERROR].includes(
+                    activeJob.jobStatus,
+                )
+            ) {
+                setIsLoading(false);
+            }
+        }
+    }, [activeJob, activeJob?.jobStatus]);
+
     if (
         user.data?.ability?.cannot('manage', 'Job') ||
         user.data?.ability?.cannot('manage', 'Project')
     )
-        return <div></div>;
+        return null;
 
     if (data?.dbtConnection?.type === DbtProjectType.NONE) {
         if (data?.dbtConnection.hideRefreshButton) {
-            return <div />;
+            return null;
         }
         return (
-            <Tooltip2
-                hoverCloseDelay={500}
-                interactionKind="hover"
-                content={
-                    <p>
+            <Popover withinPortal withArrow width={300}>
+                <Popover.Target>
+                    <Box
+                        sx={{
+                            cursor: 'pointer',
```

Comment:
so the user knows they can click the container but the button is disabled.

Reply:
I don't see the pointer cursor when the button is disabled :thinking:

Reply:
oh! Checking!

Reply:
https://github.com/lightdash/lightdash/assets/7611706/4fec0583-141c-42b9-8980-9813cfa8811b

interesting ^ I still see it...